lms/templates/ccx/student_admin.html

```diff
@@ -2,10 +2,19 @@
 
 <section>
   <h2>${_('Student Grades')}</h2>
-  <p>
-    <a href="${gradebook_url}">${_('View gradebook')}</a>
-  </p>
-  <p>
+  <p class="ccx-grade-book-download">
+    <i class="fa fa-download" aria-hidden="true"></i>
     <a href="${grades_csv_url}">${_('Download student grades')}</a>
   </p>
+  <br/>
+  <div>
+    <iframe src='${gradebook_url}' id="ccx_grade_book_iframe"
+            name="ccx_grade_book_iframe"
+            frameBorder="0"
+            scrolling="no"
+            height="360px"
+            class="ccx-grade-book">
+      <p>Iframe not supported</p>
+    </iframe>
```

Comment:
When I reviewed #10250 I verified that when you open the tab, only the first page of the gradebook is loaded. You can check the eventing and see that grade events are only fired for the learners on the first page. I am still missing what this is adding to that?